nats-concepts/what-is-nats/readme.md

```diff
@@ -1,41 +1,44 @@
 # What is NATS
 
+Software applications and services need to exchange data. NATS is an infrastructure that allows such data exchange, segmented in the form of messages. We call this a "**message oriented middleware**".
 
-NATS is what is called "message oriented middleware", meaning that it is a software infrastructure providing the exchange of data that is segmented into messages among computer applications and services.
+With NATS, application developers can:  
 
-NATS allows Application developers to easily and simply create distributed scalable client-server applications and generally store and distribute in real-time data between applications across different environments, languages, cloud providers and on-premise systems
+- Effortlessly build distributed and scalable client-server applications.
 
-### NATS Client Applications
+- Store and distribute data in realtime in a general manner. This can flexibly be achieved across various environments, languages, cloud providers and on-premises systems.
 
-Developers use one of the NATS client libraries in their application code to allow them to publish, subscribe, request and reply between instances of the application or between completely separate applications. Those applications are generally referred to as 'client Applications' or sometimes just as 'clients' (as they are clients from the NATS servers' point of view) throughout this manual.
+### NATS with Client Applications
 
-### NATS Service Infrastructure
+Developers use one of the NATS client libraries in their application code to allow them to publish, subscribe, request and reply between instances of the application or between completely separate applications. Those applications are generally referred to as 'client applications' or sometimes just as 'clients' throughout this manual (since from the point of view of the NATS server, they are clients).
+
+### NATS as Service Infrastructure
```

Comment:
Same as with NATS Client Applications, would keep it as "NATS Service Infrastructure" (no 'as')